frontend-react/src/components/AppWrapper.tsx

```diff
@@ -38,6 +39,7 @@ export const AppWrapper = ({
                                 {children}
                             </FeatureFlagProvider>
                         </AuthorizedFetchProvider>
+                        <ReactQueryDevtools initialIsOpen={false} />
```

Comment:
Does this automagically know it's in DEV, or do we need an env condition?

Reply:
It's only included when `NODE_ENV === 'development'`:

> By default, React Query Devtools are only included in bundles when process.env.NODE_ENV === 'development', so you don't need to worry about excluding them during a production build.

There's [an explicit method](https://tanstack.com/query/v4/docs/react/devtools#devtools-in-production) to include them in production, but I think having it available locally first should be the first step.